packages/stark-ui/src/modules/app-menu/components/app-menu-item.component.spec.ts

```diff
@@ -89,13 +90,15 @@ describe("StarkAppMenuItemComponent", () => {
 
 		it("should dispatch activated event", () => {
 			spyOn(component.activated, "emit");
+			spyOn(StarkComponentUtil, "isOutputWiredUp").and.returnValue(true);
```

Comment:
I suggest to take the chance to modify this test to actually use the 'activated' output of the component instead of mocking the `isOutputWiredUp`. This way it is a more realistic test in my opinion: `component.activated.subscribe(mockObserver)` because the `isOutputWiredUp` util is something used "internally" and the unit tests should focus on the functionality rather than on the implementation details.

So, instead of checking this: `expect(component.activated.emit).toHaveBeenCalledTimes(1);` we could check this:
`expect(mockObserver.next).toHaveBeenCalledTimes(1);`